test/test/helpers/nock-control/README.md

```diff
@@ -0,0 +1,29 @@
+# Nock Control
+
+## A small utility to control `nock` during a test run.
+
+We wrap all tests in the single test runner with nock to disable network access.
+On occasion, we require network access as part of a test. This package includes one
+function, `allowNetworkAccess`, which will enable network access for the duration of the describe block.
+
+## Usage
+
+```js
+import { allowNetworkAccess } from 'test/helpers/nock-control';
+
+describe('my test suite', () = {
```

Comment:
Some invalid syntax and a few style issues here:

- `() = {` should be `() => {`
- `describe('` should be `describe( '`
- `it ( 'should` should be `it( 'should`